### ISSUE TITLE
Filebeat needs to be restarted in order to send logs to logstash dynamically 

### ISSUE THREAD

**Issue by abinay** [elastic/beats#836]
Hii I was first using logstash-forwarder to ship my logs to logstash but unfortunately
logstash- forwarder was not tailing the file. It used to start reading
the file from the beginning which was not a good architectue especially
in the situation when my log file can get too large . Even after
starting logstash-forwarder with -tail = true command in the terminal
logstash-forwarder used to only harvest the file but was not processing
the events . Thats why i switched to filebeat . Now the filebeat is
tailing the file when tail_files = true is done in the configuration
file but its not pushing the logs to logstash dynamically - meaning i
have to manually restart filebeat each and everytime so as to send the
logs from filebeat to logstash .Further the window of logstash-forwarder was very
friendly as it used to tell how many events are getting processed and
all but when i run file beat using "service file beat start " it does not
let me know whether how many events it sent and it also tries only once
to connect with logstash unlike logstash-forwarder which continuously
keeps trying to connect to logstash server..............So please let me
know about this ASAP . We need to get production ready pretty soon

**Comment by urso** [comment]
Duplicate of discussion in foru,
